addons/point_of_sale/static/src/js/db.js

```diff
@@ -231,7 +231,7 @@ export const PosDB = core.Class.extend({
     add_packagings: function (product_packagings) {
         var self = this;
         _.map(product_packagings, function (product_packaging) {
-            if (_.find(self.product_by_id, { id: product_packaging.product_id[0] })) {
+            if (self.product_by_id.find(({ id }) => id === product_packaging.product_id[0])) {
```

Comment:
product_by_id is not an array. Maybe you need to do something like:

```js
Object.values(self.product_by_id).find(...)
```

Reply:
@pipu-odoo arf :-/ could you make a PR with this quick fix ?
Thanks ;-)

Reply:
@caburj do you have a testing flow ?

Reply:
I made a recording to reproduce: https://youtu.be/NrwDJtvmiz0

Reply:
done here: https://github.com/odoo/odoo/pull/117947